### PR TITLE
Use our own repository while building packages

### LINKED ISSUE #32: Install repository into build container
This allows for easier dependency management.

### LINKED ISSUE #34: init should create an empty repository
When calling `aurblobs init`, it should create an empty repository containing the database and signature files.

### PR DESCRIPTION
**Pull request by fooker**
The does the following things:
* Mounts the repository read-only while building packages (as the will be copied while signing)
* Initializes the repository with an empty database on creation (Fixes #34)
* Adds the repository as local repository to the container before updating the index (Fixes #32)

### CODE REVIEW

File: contrib/docker/init.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+gpg --import /privkey.gpg
+
+cd /repo
+
+repo-add --sign --remove $REPO_NAME.db.tar.gz $PKGS
```
Comment (mweinelt) [inline contrib/docker/init.sh]: What happens when `$PKGS` is empty?

File: aurblobs/repository.py [hunk 2]
```diff
@@ -260,7 +307,7 @@ def sign_and_add(self, pkgroot):
         for line in container.logs(stream=True):
             print('\t{0}'.format(line.decode().rstrip('\n')))
 
-        return container.wait() == 0
+        return container.wait()['StatusCode'] == 0
```
Comment (mweinelt) [inline aurblobs/repository.py]: require `docker>=3.0`

**Comment by mweinelt** [comment]
> Make repo for build read-only
> Mount repo for build read-only

File: contrib/docker/init.sh (hunk 1 above)
Comment (fooker) [reply contrib/docker/init.sh]: Copy and paste error

File: aurblobs/repository.py (hunk 2 above)
Comment (fooker) [reply aurblobs/repository.py]: Good catch. This pattern was already used before in `package.py`. Therefore i will update the requirement...

Comment (mweinelt) [reply aurblobs/repository.py]: yup, my mistake :+1:

**Pull request merged**